base_layer/core/src/consts.rs

```diff
@@ -50,3 +50,6 @@ thread_local! {
 pub const BASE_NODE_SERVICE_REQUEST_TIMEOUT: Duration = Duration::from_secs(5);
 /// The fraction of responses that need to be received for a corresponding service request to be finalize.
 pub const BASE_NODE_SERVICE_DESIRED_RESPONSE_FRACTION: f32 = 0.6;
+
+/// Maximum transaction weight used for the construction of new blocks.
+pub const MAX_BLOCK_TRANSACTION_WEIGHT: u64 = 10000; // TODO: a better weight estimate should be selected
```

Comment:
This should be a topic for an IRC dev chat, and the value should _probably_ live in `ConsensusRules`?

Reply:
Agree on this

Reply:
I moved the constant to `ConsensusRules`, and I agree that it is a topic we need to discuss.